src/api/db/structure.sql

```diff
@@ -11,7 +11,7 @@ CREATE TABLE `architectures` (
   `name` varchar(255) CHARACTER SET utf8 NOT NULL,
   `available` tinyint(1) DEFAULT '0',
   PRIMARY KEY (`id`),
-  UNIQUE KEY `arch_name_index` (`name`) USING BTREE
+  UNIQUE KEY `arch_name_index` (`name`)
```

Comment:
Please run `rake db:drop db:create db:migrate` and then commit db/structure.sql so we don't lose all these "USING BTREE" references. This is clarified in the mailing list [email](https://lists.opensuse.org/opensuse-buildservice/2017-04/msg00009.html) I sent out.